build/jobs/provision-deploy.yml

```diff
@@ -22,9 +22,6 @@ parameters:
 - name: schemaAutomaticUpdatesEnabled
   type: string
   default: 'tool'
-- name: sqlServerAdminPassword
```

Comment:
I think we can remove the "UpdateRandom" step too since I don't think anything else uses it

Reply:
Should we delete the update-sqlAdminPassword.yml file as well since it will no longer be used?